app/assets/onepager/css/main.css

```diff
@@ -560,6 +560,18 @@ body #main a:not(.button):hover {
   margin: auto;
 }
 
+.alert {
+  position: absolute;
```

Comment:
why this one? is not already absolute positioned?

Reply:
Not sure am the one that introduced this. But am going to remove it now

Reply:
pleaser remove this alert rules are already in base https://gitcoin.co/tip/send/2 you can check there if you disable metamask is fixed on top and also have rules to handle multiple alerts